spec/unit/page_spec.rb

```diff
@@ -61,7 +61,9 @@ def config(options = {})
 
     describe "#namespace_name" do
       it "returns the name of the namespace" do
-        expect(config.namespace_name).to eq "admin"
+        ActiveSupport::Deprecation.silence do
+          expect(config.namespace_name).to eq "admin"
```

Comment:
This suggests something needs to be fixed in page.rb.  You still need to do pages for nested namespaces, right?